src/absil/illex.fsl

```diff
@@ -16,7 +16,8 @@ open FSharp.Compiler.AbstractIL.Internal.AsciiParser
 open FSharp.Compiler.AbstractIL.Internal.AsciiConstants 
 
 
-let lexeme (lexbuf : LexBuffer<char>) = new System.String(lexbuf.Lexeme)
+let lexeme (lexbuf : LexBuffer<char>) = LexBuffer<char>.LexemeString lexbuf
+let lexeme_char (lexbuf : LexBuffer<char>) n = lexbuf.LexemeChar n
```

Comment:
nit: rename to `lexemeChar` as we normally do camel-casing.

Reply:
ok, how do you feel about calling `lexbuf.LexemeChar` directly in the code instead of a helper?

Reply:
That would be fine.